idea/frontend/src/shared/config/consts.ts

```diff
@@ -6,6 +6,8 @@ import { isAccountAddressValid } from '../helpers';
 const API_URL = import.meta.env.VITE_API_URL as string;
 const INDEXER_API_URL = import.meta.env.VITE_INDEXER_API_URL as string;
 const NODES_API_URL = import.meta.env.VITE_NODES_API_URL as string;
+const FAUCET_API_URL = import.meta.env.VITE_FAUCET_API_URL as string;
+const META_STORAGE_API_URL = import.meta.env.VITE_META_STORAGE_API_URL as string;
```

Comment:
```suggestion
const METADATA_STORAGE_API_URL = import.meta.env.VITE_METADATA_STORAGE_API_URL as string;
```

Also, can be moved to metadata feature.

Reply:
This url is used by metadata and sails features. So it's best to leave it here.